File: DayOne/pythonPhase_Gate/NumberGuessingGame.py
#1. Import the random module
#2. Collect userInput
#3. validate user input
#4. ----
import random

print("=" * 70)
print("=" * 70)

print("""
   Welcome To Mini Number Guessing Game.

   Pick a random number between 1 to 100.

   You have 5 attempt to guess the number.

   """)

while True:

  numberGuessed = 0;
  numberOfAttempt = 5;
  rating = 0;
  correctNumber = numberToGuess;

  userInput = int(input("Enter A Number: "))

  numbers_to_guess = random.randint(1, 100)

  print(numbers_to_guess)

File: DayOne/javaPhase_gate/NumberGuessingGame.java
public class NumberGuessingGame {
  public static void main(String[] args) {

    java.util.Random random = new java.util.Random();
    java.util.Scanner inputCollector = new java.util.Scanner(System.in);

    System.out.println(" ** W.E.L.C.O.M.E  T.O  N.U.M.B.E.R  G.U.E.S.S.I.N.G  ** ");
    System.out.println("Welcome To Mini Number Guessing Game.\nPick a random number between 1 to 100.\nYou have 5 attempt to guess the number.");

    while(true) {
    int numberToGuess = random.nextInt(100) + 1;

    System.out.print("\nEnter Number: ");
    int userInput = inputCollector.nextInt();
    if(userInput == -1) {
      System.out.println("Terminated");
      break;
    }

    int numberGuessed = 0;
    int numberOfAttempt = 5;
    int rating = 0;
    int correctNumber = numberToGuess;

    if(userInput < numberToGuess || userInput > 100) {
      System.out.println("Too High, Enter Number between 1 to 100");
      numberOfAttempt--;
    }
    else if(userInput == numberToGuess) {
      System.out.println("YOU WIN!!");
      break;

    } else if(userInput < numberToGuess) {
        System.out.println("Too Low");
	numberOfAttempt--;
	numberGuessed++;

    } else if(userInput > numberToGuess) {
        System.out.println("Too High");
	numberGuessed++;
	numberOfAttempt--;
    }
    System.out.println("The correct number is: " + correctNumber);
    }// end of while true braces
  }// end of main meth0d braces
}// end of class brace
